src/openrct2/drawing/ImageImporter.h

```diff
@@ -70,12 +74,12 @@ namespace OpenRCT2::Drawing
 
         static std::vector<int32_t> GetPixels(
             const uint8_t* pixels, uint32_t pitch, uint32_t srcX, uint32_t srcY, uint32_t width, uint32_t height,
-            Palette palette, ImportFlags flags, ImportMode mode);
+            Palette palette, ImportFlags flags, ImportMode mode, GamePalette spritePalette);
```

Comment:
You are passing this function two palettes that makes no sense. It should only take the new palette (const ref). You will need to modify the caller's a bit to handle this